docs/pt/docs/project-generation.md

```diff
@@ -0,0 +1,84 @@
+    * <a href="https://fastapi.tiangolo.com/features/" class="external-link" target="_blank">**Muitos outros recursos**</a> incluindo validação automática, serialização, documentação interativa, autenticação com _tokens_ OAuth2 JWT etc.
+* **Senha segura** _hashing_ por padrão.
+* Autenticação **Token JWT**.
+* Modelos **SQLAlchemy** (independente de extensões Flask, para que eles possam ser usados com _workers_ Celery diretamente).
+* Modelos básicos para usuários (modifique e remova conforme suas necessidades).
+* Migrações **Alembic**.
+* **CORS** (_Cross Origin Resource Sharing_ - Compartilhamento de Recursos Entre Origens).
+* _Worker_ **Celery** que pode importar e usar modelos e códigos do resto do _backend_ seletivamente.
+* Testes _backend_ _REST_ baseados no **Pytest**, integrados com Docker, então você pode testar a interação completa da API, independente do banco de dados. Como roda no Docker, ele pode construir um novo repositório de dados do zero toda vez (assim você pode usar ElasticSearch, MongoDB, CouchDB, ou o que quiser, e apenas testar que a API esteja funcionando).
+* Fácil integração com Python através dos **Kernels Jupyter** para desenvolvimento remoto ou no Docker com extensões como Atom Hydrogen ou Visual Studio Code Jupyter.
+* _Frontend_ **Vue**:
+    * Gerado com Vue CLI.
+    * Controle de **Autenticação JWT**.
+    * Visualização de _login_.
+    * Após o _login_, visualização do painel de controle principal.
+    * Painel de controle principal com criação e edição de usuário.
+    * Edição do próprio usuário.
+    * **Vuex**.
+    * **Vue-router**.
+    * **Vuetify** para belos componentes _material design_.
+    * **TypeScript**.
+    * Servidor Docker baseado em **Nginx** (configurado para rodar "lindamente" com Vue-router).
+    * Construção multi-estágio Docker, então você não precisa salvar ou _commitar_ código compilado.
+    * Testes _frontend_ rodados na hora da construção (pode ser desabilitado também).
+    * Feito tão modular quanto possível, então ele funciona fora da caixa, mas você pode gerar novamente com Vue CLI ou criar conforme você queira, e reutilizar o que quiser.
+* **PGAdmin** para banco de dados PostgreSQL, você pode modificar para usar PHPMyAdmin e MySQL facilmente.
+* **Flower** para monitoração de tarefas Celery.
+* Balanceamento de carga entre _frontend_ e _backend_ com **Traefik**, então você pode ter ambos sob o mesmo domínio, separados por rota, mas servidos por diferentes containers.
+* Integração Traefik, incluindo geração automática de certificados **HTTPS** Let's Encrypt.
+* GitLab **CI** (integração contínua), incluindo testes _frontend_ e _backend_.
+
+## Full Stack FastAPI Couchbase
+
+GitHub: <a href="https://github.com/tiangolo/full-stack-fastapi-couchbase" class="external-link" target="_blank">https://github.com/tiangolo/full-stack-fastapi-couchbase</a>
+
+⚠️ **WARNING** ⚠️
```

Comment:
Será que eeria interessante traduzir warning pra atenção? Ou já é um padrão?

Reply:
Good point!

I'm not sure, I would imagine yes, but as there are some words that work better in English (even in Portuguese), I'll leave it like this for now. And we can update it later if there's a consensus on that in the Portuguese Translations issue: https://github.com/tiangolo/fastapi/issues/1211

But meanwhile, it should be fine and we can have this merged :rocket: